src/opera/pge/cslc_s1/cslc_s1_pge.py

```diff
@@ -619,7 +619,12 @@ def _collect_cslc_product_metadata(self, metadata_product):
             use with the ISO metadata Jinja2 template.
 
         """
-        output_product_metadata = get_cslc_s1_product_metadata(metadata_product)
+        # Extract all metadata assigned by the SAS at product creation time
+        try:
+            output_product_metadata = get_cslc_s1_product_metadata(cslc_product)
+        except Exception as err:
+            msg = f'Could not extract metadata from {cslc_product}: Exception {err}'
```

Comment:
This is nit-picky for sure, but for the sake of uniformity I would like these error messages to be formatted in a way that aligns with other logged errors. Something like:

```
msg = f'Failed to extract metadata from {cslc_product}, reason: {err}'
```